srcpkgs/gotify-cli/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'gotify-cli'
+pkgname=gotify-cli
+version=2.2.1
+revision=1
+wrksrc="cli-$version"
+build_style=go
+go_import_path="github.com/gotify/cli/v2"
```

Comment:
go_ldflags are missing:
```
%: gotify-cli version
Version:   unknown
Commit:    unknown
BuildDate: unknown
```

Reply:
> go_ldflags are missing:
> 
> ```
> %: gotify-cli version
> Version:   unknown
> ->Commit:    unknown
> BuildDate: unknown
> ```

Do we need that Commit line? It would have to be manually fetched and updated in the template on each update.

Reply:
just Version

Reply:
Is how it is now fine? I also included `main.BuildDate`, but can remove it. The only other go template I saw including that was gocryptfs.